HousingFinanceInterimApi/V1/Domain/AdjustmentDomain.cs

```diff
@@ -11,26 +11,17 @@ public class AdjustmentDomain
     {
         public int Id { get; set; }
 
-        [JsonProperty("Tenancy Agreement Ref")]
-        public string TenancyAgreementRef { get; set; }
-
-        [JsonProperty("Year")]
-        public int Year { get; set; }
-
-        [JsonProperty("Period (Week/Month)")]
-        public int Period { get; set; }
+        [JsonProperty("Payment Ref")]
```

Comment:
Please confirm, is this the PRN number or the Tag Ref?  If so, is this using the payment ref instead of the tag ref?

Reply:
Yes, this is the PRN number. The users have the PRN number instead of tag_ref, mainly for the new tenancies. Internally the system finds the tag_ref using the PRN number.